index.js

```diff
@@ -13,6 +8,10 @@ function mewt (target) {
   }
 
   const override = prop => (...args) => {
+    const multiRet = 'push pop shift unshift'
```

Comment:
What is the purpose of moving these here? The interpreter probably optimizes this, but still it doesn't look right to me. I'd rather move these out of the functions scope, but then the minifier doesn't minify variable names.